R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -3613,11 +3613,11 @@ test_that("Collect on DataFrame when NAs exists at the top of a timestamp column
 test_that("catalog APIs, currentDatabase, setCurrentDatabase, listDatabases", {
   expect_equal(currentDatabase(), "default")
   expect_error(setCurrentDatabase("default"), NA)
-  expect_error(setCurrentDatabase("foo"),
-               "Error in setCurrentDatabase : analysis error - Database 'foo' does not exist")
+  expect_error(setCurrentDatabase("zxwtyswklpf"),
+        "Error in setCurrentDatabase : analysis error - Database 'zxwtyswklpf' does not exist")
   dbs <- collect(listDatabases())
   expect_equal(names(dbs), c("name", "description", "locationUri"))
-  expect_equal(dbs[[1]], "default")
+  expect_equal(which(dbs[, 1] == "default"), 1)
```

Comment:
I wonder if there is a better way to ensure the default database is named "default", perhaps? this checks for "exactly one database is named "default"" - I guess that's ok...

Reply:
@felixcheung Thanks for reviewing. Actually the name "default" for default database is fixed, to the best of my knowledge. Its even hardcoded [here](https://github.com/apache/spark/blob/c3be2cd347c42972d9c499b6fd9a6f988f80af12/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala#L45-47). So we should be okay with this check. Please let me know what you think.